connectors/twitter-mention-connector/src/main/resources/camel-connector.json

```diff
@@ -2,7 +2,7 @@
   "baseScheme" : "twitter",
   "baseGroupId" : "org.apache.camel",
   "baseArtifactId" : "camel-twitter",
-  "baseVersion" : "2.19.0.fuse-000031",
+  "baseVersion" : "2.19.0.fuse-000033",
```

Comment:
yeah, have to wait until 000035 is there so that I can build. These are generated files.

will push then asap.